torch/csrc/jit/codegen/cuda/lower_instrument.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include <torch/csrc/jit/codegen/cuda/ir_all_nodes.h>
+
+namespace torch {
+namespace jit {
+namespace fuser {
+namespace cuda {
+
+//! Set up kernel profiling when enabled
```

Comment:
Add some more to the comment here describing the pass a bit please. Some good notes to add:
New Exprs are returned as normal lowering passes tend to do. The buffer allocation to record profiling information is added to the beginning of the Exprs. However the big purpose of this lowering pass is to fill the information about the Expr's to be profiled in KernelPerformanceProfile which escapes this scope through the current GpuLower object.

Reply:
Thanks. Added more comments.